core/src/test/java/org/infinispan/commons/util/concurrent/jdk8backported/BoundedEquivalentConcurrentHashMapV8BaseTest.java

```diff
@@ -32,6 +9,13 @@
 import org.infinispan.util.logging.LogFactory;
 import org.testng.annotations.Test;
 
+import java.util.Arrays;
+import java.util.Map;
```

Comment:
Yeah I believe I wrote this class using Eclipse? Either way Eclipse and IntelliJ like to put them in different order.